IO/PROTOCOLS/SCPI/MEASURE_VOLTAGE/MEASURE_VOLTAGE.py

```diff
@@ -0,0 +1,49 @@
+import serial
+import traceback
+from flojoy import flojoy, SerialConnection, TextBlob, Scalar
+from typing import cast, Optional
+
+
+@flojoy(deps={"pyserial": "3.5"}, inject_connection=True)
+def MEASURE_VOLTAGE(
+    connection: SerialConnection, default: Optional[TextBlob] = None
```

Comment:
Should probably just take any datacontainer type since it's not being used: 

`default: Optional[DataContainer] = None,`